setup.py

```diff
@@ -51,24 +51,33 @@
     CMDCLASS = {}
 else:
     EXTENSIONS = {
-        'core': ['expression', 'numbers', 'rules', 'pattern'],
-        'builtin': ['arithmetic', 'numeric', 'patterns', 'graphics']
+        "core": ["expression", "numbers", "rules", "pattern"],
+        "builtin": ["arithmetic", "numeric", "patterns", "graphics"],
     }
     EXTENSIONS = [
-        Extension('mathics.%s.%s' % (parent, module),
-                  ['mathics/%s/%s.py' % (parent, module)])
-        for parent, modules in EXTENSIONS.items() for module in modules]
-    CMDCLASS = {'build_ext': build_ext}
-    INSTALL_REQUIRES += ['cython>=0.15.1']
+        Extension(
+            "mathics.%s.%s" % (parent, module), ["mathics/%s/%s.py" % (parent, module)]
+        )
+        for parent, modules in EXTENSIONS.items()
+        for module in modules
+    ]
+    CMDCLASS = {"build_ext": build_ext}
+    INSTALL_REQUIRES += ["cython>=0.15.1"]
 
 # General Requirements
-INSTALL_REQUIRES += ['sympy>=1.6, < 1.7', 'django >= 1.8, < 1.12',
-                     'mpmath>=1.1.0', 'python-dateutil', 'colorama']
+INSTALL_REQUIRES += [
+    "sympy>=1.6, < 1.7",
+    "django >= 1.8, < 1.12",
+    "llvmlite",
```

Comment:
I see. This is the cause of the issue.

Reply:
@rocky Why did you add `llvmlite` to the install dependencies?

Reply:
That was in the last force push. It to try to addresss the `..so` not getting found by the previous commit.